tests/unit/other.tcl

```diff
@@ -24,6 +24,12 @@ start_server {tags {"other"}} {
         }
     }
 
+    test {MEMORY PURGE basic} {
```

Comment:
let's add the word "coverage" here.
please go over all the changes in the tests and make sure it's consistent.
i.e. i want the reader of that test to understand why it's here (considering the fact it doesn't really test anything)